File: speed/fib.cs
using System;
public class Fib {
	public static int fib (int n) {
		if (n < 2) return n;
		return fib(n-2)+fib(n-1);
	}
	public static int Main (string[] args) {
		Console.WriteLine(fib(32));
		return 0;
	}
}

File: speed/fib.go
package main
import "fmt"
func fib(n uint) uint {
    if n == 0 {
        return 0
    } else if n == 1 {
		return 1
	} else {
		return fib(n-1) + fib(n-2)
	}
}
func main() {
	n := uint(42)
	fmt.Println(fib(n))
}
